qiskit_aer/noise/__init__.py

```diff
@@ -66,10 +66,12 @@
     circ.measure([0, 1, 2], [0, 1, 2])
 
     # Perform a noise simulation
-    result = execute(circ, Aer.get_backend('qasm_simulator'),
-                     coupling_map=coupling_map,
-                     basis_gates=basis_gates,
-                     noise_model=noise_model).result()
+    backend = Aer.get_backend('qasm_simulator')
```

Comment:
Could you define `backend` as

```
backend = AerSimulator(noise_model=noise_model,
                       coupling_map=coupling_map,
                       basis_gates=basis_gates)
```

instead of `Aer.get_backend('qasm_simulator')`?

Transpilation will be more simple as `transpiled_circuit = transpile(circ, backend)`

Reply:
ok